pkg/mcpserver/server.go

```diff
@@ -53,3 +55,27 @@ func RunServer(mcpServer *mcpfile.MCPServer) error {
 		return fmt.Errorf("tried running invalid transport protocol")
 	}
 }
+
+// RunServers runs all servers defined in the MCP file
+func RunServers(mcpFilePath string) error {
```

Comment:
Could you use this method in https://github.com/genmcp/gen-mcp/blob/2d1e77ac3766b985a20365b7672722279bc33d39/cmd/genmcp-server/main.go#L12 as well?

Reply:
:facepalm: sure. Slipped somehow...